scipy/sparse/dia.py

```diff
@@ -72,6 +72,21 @@ class dia_matrix(_data_matrix):
            [0, 2, 3, 0],
            [0, 0, 3, 4]])
 
+    >>> import numpy as np
```

Comment:
This import (and only this one) is impicit in all scipy docstring examples, so this line should be removed.

Reply:
Hi @ev-br, Thanks for review this, it seems all checks have passed now.